packages/fxa-auth-server/package.json

```diff
@@ -39,10 +39,10 @@
     "audit-orphaned-stripe-accounts": "CONFIG_FILES='config/secrets.json' node -r esbuild-register ./scripts/audit-orphaned-customers.ts",
     "remove-unverified-accounts": "CONFIG_FILES='config/secrets.json' node -r esbuild-register ./scripts/remove-unverified-accounts.ts",
     "emails-scss": "node -r esbuild-register ./lib/senders/emails/sass-compile-files.ts",
-    "storybook": "npm run install-ejs && yarn emails-scss && NODE_OPTIONS=--openssl-legacy-provider start-storybook -p 6010 --no-version-updates -s ./",
-    "build-storybook": "npm run install-ejs && yarn emails-scss && NODE_OPTIONS=--openssl-legacy-provider build-storybook && yarn build-storybook:copy-locales && yarn build-storybook:copy-templates",
-    "build-storybook:copy-locales": "mkdir -p ./storybook-static/public/locales && cp -R ./public/locales ./storybook-static/public",
-    "build-storybook:copy-templates": "mkdir -p ./storybook-static/lib/senders/emails/templates && cp -R ./lib/senders/emails ./storybook-static/lib/senders",
+    "storybook": "yarn l10n-prime && npm run install-ejs && yarn emails-scss && NODE_OPTIONS=--openssl-legacy-provider start-storybook -p 6010 --no-version-updates -s ./",
+    "build-storybook": "yarn l10n-prime && npm run install-ejs && yarn emails-scss && NODE_OPTIONS=--openssl-legacy-provider build-storybook && yarn build-storybook:copy-locales && yarn build-storybook:copy-templates",
```

Comment:
I'm a bit confused here&hellip;

When we run <kbd>yarn build-storybook</kbd>, we'll call <kbd>yarn l10n-prime</kbd>.
But also this script calls <kbd>yarn build-storybook:copy-locales</kbd> (which also calls <kbd>yarn l10n-prime</kbd>) as well as <kbd>yarn build-storybook:copy-templates</kbd> (which also also calls <kbd>yarn l10n-prime</kbd>).

So does calling `yarn build-storybook` end up [in]directly calling `yarn l10n-prime` 3&times;?